python/arcticdb/__init__.py

```diff
@@ -10,8 +10,16 @@
 from arcticdb.tools import set_config_from_env_vars
 from arcticdb_ext.version_store import DataError, VersionRequestType
 from arcticdb_ext.exceptions import ErrorCode, ErrorCategory
-from arcticdb.version_store.library import WritePayload, ReadInfoRequest, ReadRequest
-from arcticdb.version_store.library import StagedDataFinalizeMethod, WriteMetadataPayload
+from arcticdb.version_store.library import (
+    WritePayload,
+    ReadInfoRequest,
+    ReadRequest,
+    col,
```

Comment:
perhaps a test for the new "col" feature? to show how to use it.

Reply:
That's covered here:
https://github.com/man-group/ArcticDB/blob/71d917621b1aa9360df1c87fa3d426c4df592f80/python/tests/unit/arcticdb/version_store/test_lazy_dataframe.py#L123